ten_quick_tips_live_coding.md

```diff
@@ -56,6 +56,14 @@ instructors will often make unplanned mistakes.
 Novice learners are likely to make many such mistakes themselves,
 and diagnosing and solving mistakes is an integral aspect of learning programming.
 
+The participatory, 'code-along' aspect
```

Comment:
A few different points are being made here: the target audience (novice, experienced) matters, how they engage (passive versus active) matters. Suggest rewording and elaborating. Something like... 

The participatory aspect engages learners, which helps them become active practitioners rather than passive observers of the programming process. Participatory live coding is most beneficial for novices who are unfamiliar with the tools. More experienced learners may gain enough by listening passively or engaging with the material and classroom differently.